platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -180,26 +177,24 @@ function UpgradeFW() {
 
     if [ ! -z "${_FS_MOUNTPOINT}" ]; then
         local -r _FW_FILE="${_FS_MOUNTPOINT}/${FW_FILE_MAP[$_ASIC_TYPE]}"
-
-        if [ ! -f "${_FW_FILE}" ]; then
-            ExitFailure "no such file: ${_FW_FILE}"
-        fi
-
-        RunCmd "${QUERY_CMD} -i ${_FW_FILE} -L ${QUERY_FILE}" &>/dev/null
-
-        local -r _FW_INFO="$(grep FW ${QUERY_FILE})"
-        local -r _FW_CURRENT="$(echo ${_FW_INFO} | cut -f2 -d' ')"
-        local -r _FW_AVAILABLE="$(echo ${_FW_INFO} | cut -f3 -d' ')"
     else
         local -r _FW_FILE="${FW_FILE_MAP[$_ASIC_TYPE]}"
+    fi
 
-        RunCmd "${QUERY_CMD} -L ${QUERY_FILE}" &>/dev/null
-
-        local -r _FW_INFO="$(grep FW ${QUERY_FILE})"
-        local -r _FW_CURRENT="$(echo ${_FW_INFO} | cut -f2 -d' ')"
-        local -r _FW_AVAILABLE="${FW_REQUIRED_MAP[$_ASIC_TYPE]}"
+    if [ ! -f "${_FW_FILE}" ]; then
+        ExitFailure "no such file: ${_FW_FILE}"
     fi
 
+    RunCmd "${QUERY_CMD} -L ${QUERY_FILE}" &>/dev/null
+    local -r _FW_INFO="$(grep FW ${QUERY_FILE})"
+    local -r _PSID_INFO="$(grep PSID ${QUERY_FILE})"
+    local -r _PSID="$(echo ${_PSID_INFO} | cut -f2 -d' ')"
+    local -r _FW_CURRENT="$(echo ${_FW_INFO} | cut -f2 -d' ')"
+
+    RunCmd "${LIST_CONTENT_CMD} -i ${_FW_FILE} > ${LIST_CONTENT_FILE}"
```

Comment:
@ayurkiv-nvda suggest to have `${LIST_CONTENT_CMD} -i ${_FW_FILE} -o ${LIST_CONTENT_FILE}`

Reply:
Done